userspace/libscap/scap_savefile.c

```diff
@@ -1165,6 +1170,28 @@ static int32_t scap_read_proclist(scap_t *handle, gzFile f, uint32_t block_lengt
 		totreadsize += readsize;
 
 		//
+		// exepath
+		//
+		readsize = gzread(f, &(stlen), sizeof(uint16_t));

```

Comment:
I have followed the code and added PL_BLOCK_TYPE_V7, hope that I have understood the usage and didn't forget anything.
Let me know how it is looking now.